auto_nag/scripts/workflow/no_severity.py

```diff
@@ -35,7 +36,9 @@ def __init__(self, typ, inactivity_days: int = 3):
         )
         self.round_robin = RoundRobin.get_instance()
         self.components_skiplist = utils.get_config("workflow", "components_skiplist")
-        self.activity_date = lmdutils.get_date("today", inactivity_days)
+        self.activity_date = numpy.busday_offset(
+            lmdutils.get_date("today"), -inactivity_days
+        )
```

Comment:
```suggestion
        self.activity_date = str(
            numpy.busday_offset(lmdutils.get_date("today"), -inactivity_days)
        )
```
Nit: By converting it here to a string we can avoid later covert all dates to `datetime64` (L83).

Reply:
Fixed, thanks!